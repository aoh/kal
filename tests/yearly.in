every year on 3.1.: event
