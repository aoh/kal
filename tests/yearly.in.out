3.1.1970, Saturday, week 1
 - event

every year on 3.1.: event
